backends-clickhouse/src/main/scala/org/apache/spark/sql/execution/datasources/clickhouse/utils/MergeTreePartsPartitionsUtil.scala

```diff
@@ -377,6 +377,9 @@ object MergeTreePartsPartitionsUtil extends Logging {
       partNameWithLocation: util.HashMap[String, String],
       locationDistinct: util.HashSet[String]): Unit = {
 
+    logWarning(
+      s"[DEBUG] PR-8245 " +
+        s"partNameWithLocation = $partNameWithLocation,  locationDistinct = $locationDistinct")
```

Comment:
Looks like this is a debug log? Can we remove it?

Reply:
thanks.